### ISSUE TITLE
aql query is appending the include properties everytime to the query

### ISSUE THREAD

**Issue by pinguxx** [jnszkr/aql#4]
we are doing this query
```
aql.items
            .find({
                type: 'folder',
                repo: repo,
                depth: 1
            })
            .include('name', 'repo', 'path')
            .sort({ $asc: ['name'] })
```

but after running that several times, we got an error with this body
```
items.find({"type":"folder","repo":"dev-deploy","depth":1}).include("name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path", "name", "repo", "path").sort({"$asc":["name"]})
```

is there something we can call to clear the properties, for the code seems that is concatenating everytime
https://github.com/jnszkr/aql/blob/e8ca97ffacc484caff9c8429c87f940f91049ccc/aql.js#L62

**Comment by jnszkr** [comment]
Hi,

Thanks for reporting this issue.

Ideally, new DomainQuery object is created every time you call `aql.items`, see

https://github.com/jnszkr/aql/blob/e8ca97ffacc484caff9c8429c87f940f91049ccc/aql.js#L171-L173

Therefore, I am not sure how you could get this result.

Anyway, I will add a fix that prevents duplicates.

BR

**Comment by jnszkr** [comment]
fixed v1.0.4
